Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
       railties (>= 3.1.1)
-    trestle (0.9.3)
+    trestle (0.9.4)
       activemodel (>= 4.2.0)
       kaminari (>= 1.1.0)
       railties (>= 4.2.0)
       sprockets-rails (>= 2.0.0)
-    trestle-active_storage (3.0.0)
+    trestle-active_storage (3.0.1)
       rails (>= 5.2, < 7)
       trestle (~> 0.9.0, >= 0.9.3)
-    trestle-auth (0.4.0)
+    trestle-auth (0.4.1)
       bcrypt (~> 3.1.7)
-      trestle (~> 0.9.0)
+      trestle (~> 0.9.0, >= 0.9.3)
     trestle-tinymce (0.3.0)
       tinymce-rails (~> 5.1)
       trestle (~> 0.9.0)
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.7)
+    tzinfo (1.2.8)
       thread_safe (~> 0.1)
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.7.7)
-    warden (1.2.8)
-      rack (>= 2.0.6)
-    web-console (4.0.4)
+    warden (1.2.9)
```

Comment:
### [warden](https://github.com/hassox/warden)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.8 → 1.2.9

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/wardencommunity/warden/compare/v1.2.8...v1.2.9).

These are the first 10 commits:
- (e0c3b41) [change warden wiki link in README.md](https://github.com/wardencommunity/warden/commit/e0c3b413acb658c5b40dc2ab04c0e23ee3463d04)
- (ef0b1f4) [Merge pull request #172 from buncismamen/master](https://github.com/wardencommunity/warden/commit/ef0b1f4f44d7cc0e738b7e40c049d89e60351a51)
- (ed160f6) [Fix document comment[ci skip]](https://github.com/wardencommunity/warden/commit/ed160f6738d8a00333c9517e3f5170321a13566a)
- (902903a) [Add newer ruby versions for CI](https://github.com/wardencommunity/warden/commit/902903a12bd261809ce8e59fa59e8e031765349c)
- (fdb7189) [Merge pull request #174 from colorbox/add_ruby_version_on_CI](https://github.com/wardencommunity/warden/commit/fdb71899c8c85f25e2259e957edbdd15dfe36024)
- (df575ff) [Merge branch 'master' into fix_document](https://github.com/wardencommunity/warden/commit/df575ff21c7a3598043e0a67daafb727482deb80)
- (b7ff0f4) [Merge pull request #173 from colorbox/fix_document](https://github.com/wardencommunity/warden/commit/b7ff0f4ffacd7ad5a9b5a0191f1da0c7a64b2c2c)
- (c1af24f) [Set `winning_strategy` when `:warden` is thrown](https://github.com/wardencommunity/warden/commit/c1af24f556b5faf4f7f1acfe49a02f56b3c5c567)
- (7aec137) [Merge pull request #175 from landongrindheim/set-winning-str…](https://github.com/wardencommunity/warden/commit/7aec1374cba3409d82208c5c7fb0c98f18a0c2c1)
- (3110169) [gemspec: rubyforge_project is removed](https://github.com/wardencommunity/warden/commit/31101697acc19aa7f6bf3f07f6d462eb7446fa63)


</details>